index.html

```diff
@@ -11,18 +11,29 @@
     <header>
       <nav>
         <a href="#" class="nav-h3">Welcome</a>
-        <img
+        <!-- <img
           src="https://img.icons8.com/metro/18/000000/menu.png"
           alt="menu icon"
-        />
+        /> -->
+        <ul class='Navmenu'>
```

Comment:
### To match the Figma design:
- [ ] Kindly change the color of the close button.
- [ ] Please increase the right spacing between the close button and the end of page.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/195317346-851da004-cd5e-4dd6-ada1-17119bf482e3.png) | ![image](https://user-images.githubusercontent.com/24703171/195316747-edf1a460-2406-4a30-aaba-f15bd05a42e2.png) |